### ISSUE TITLE
Animation - bones scale keys exported?

### ISSUE THREAD

**Issue by nonameboy** [Ploaj/StudioSB#30]
Hi. When importing animation into blender I think for some animations, the scale values of bones are not exported. It might be something else, I dont know. See gif
https://i.imgur.com/JWQK5gk.gif

**Comment by Ploaj** [comment]
What format did you export? SMD doesn't support scale.

**Comment by nonameboy** [comment]
Yes tried smd. Also importing .nuanmb same result. I'm using this addon in blender SSBUlt_NUANMB.py
